### PR TITLE
RTS.unsafeToFuture / IO.fromFuture

### PR DESCRIPTION
**Pull request by fommil**
moving #45 with some cleanups

### CODE REVIEW

File: ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala [hunk 1]
```diff
@@ -42,6 +43,20 @@ trait RTS {
     }
   }
 
+  /**
+   * WARNING: UNSAFE - convert between IO[?, A] and Future - do not use this method.
+   * Seriously, try not to.
+   */
+  final def unsafeToFuture[E, A](io: IO[E, A]): concurrent.Future[E \/ A] = {
```
Comment (hrhino) [inline ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala:50]: your lack of naming conviction is unfortunate.

File: ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala [hunk 2]
```diff
@@ -42,6 +43,20 @@ trait RTS {
     }
   }
 
+  /**
+   * WARNING: UNSAFE - convert between IO[?, A] and Future - do not use this method.
+   * Seriously, try not to.
```
Comment (hrhino) [inline ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala:48]: .....

File: ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala (hunk 1 above)
Comment (fommil) [reply ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala:50]: I wanted to! But this makes writing scalafix rules easier...

Comment (hrhino) [reply ioeffect/src/main/scala/scalaz/ioeffect/RTS.scala:50]: are you just forbidding methods named `unsafe*`?

**Pull request merged**